python/tk_multi_loader/dialog.py

```diff
@@ -1180,16 +1201,26 @@ def _add_rdo_list_view_mod(self):
             self.ui.label_2.hide()
             self.ui.thumb_scale.hide()
 
-    def _add_rdo_publish_search(self):
+    def _add_rdo_status_filter(self, hlayout):
+        '''
+        Add a combo box to sort the latest publishes by status
+        Kind of copying off the _add_rdo_status_filter
+        '''
+        filter_label = QtGui.QLabel("Filter by Status")
```

Comment:
And it gives us the Type filter on the lower left of the window and Status filter at the center top.

So I guess it would be cleaner to have a filter panel similar to the one in Shotgun:
![sg_filter_panel](https://cloud.githubusercontent.com/assets/2866068/8968526/cca075c2-360b-11e5-9e54-208bb09e5abb.png)

But we'll make another ticket for that.